python/ray/client_builder.py

```diff
@@ -77,7 +81,16 @@ def _split_address(address: str) -> Tuple[str, str]:
 
 
 def _get_builder_from_address(address: Optional[str]) -> ClientBuilder:
-    if address is None or address == "local":
+    if address == "local":
+        return _LocalClientBuilder(None)
+    if address is None:
+        try:
+            with open("/tmp/ray/current_cluster", "r") as f:
+                address = f.read()
+                print(address)
+        except FileNotFoundError:
+            # `address` won't be set and we'll create a new cluster.
+            pass
```

Comment:
why not to check if the file exists and then to read it?  Also, why are we printing the address here?